src/ripple/app/misc/impl/Manifest.cpp

```diff
@@ -369,7 +384,7 @@ ManifestCache::load (
             beast::detail::base64_decode(configManifest));
         if (! mo)
         {
-            JLOG (j_.error()) << "Malformed manifest in config";
+            JLOG (j_.error()) << "Malformed token manifest in config";
```

Comment:
I wanted to mention "token" because this message is the result of a malformed entry in the `[validator_token]` field.